cmd/agent/dist/conf.d/systemd.d/conf.yaml.example

```diff
@@ -0,0 +1,33 @@
+init_config:
+
+instances:
+    -
+    ## @param unit_names - list of strings - optional
+    ## List of systemd units to monitor (full name e.g. ssh.service).
+    ## If unit_names and unit_regexes are both NOT specified, the check will monitor all units.
+    #
+    # unit_names:
+    #   - <UNIT_NAME>
+
+    ## @param unit_regexes - list of strings - optional
+    ## Regex pattern[s] matching the names of units to monitor.
+    ## If unit_names and unit_regexes are both NOT specified, the check will monitor all units.
+    ## 
+    ## If the number of units managed by systemd is unbounded, it might infinitely increase
+    ## the number of metrics.
+    ## Learn more about custom metrics: https://docs.datadoghq.com/developers/metrics/custom_metrics/#how-is-a-custom-metric-defined
+    #
+    # unit_regexes:
+    #   - <UNIT_NAME>          # match any unit name containing "<UNIT_NAME>"
+    #   - ^<UNIT_NAME>$        # match exactly the unit name "<UNIT_NAME>"
+    #   - ^<PREFIX>            # match unit name with prefix "<PREFIX>"
+    #   - <SUFFIX>$            # match unit name with suffix "<SUFFIX>"
+
+    ## @param tags  - list of key:value elements - optional
+    ## List of tags to attach to every metric, event, and service check emitted by this integration.
+    ##
+    ## Learn more about tagging at https://docs.datadoghq.com/tagging
```

Comment:
Nit: Since this is a standard option I think it might make sense to always have it at the top

Reply:
Actually, in `integrations-core`, most integrations have `tags` config at the bottom.

Reply:
Ok! As long as it's consistent all good 👍